elytron/src/main/java/org/wildfly/extension/elytron/AdvancedModifiableKeyStoreDecorator.java

```diff
@@ -125,6 +145,18 @@
             .setRequired(true)
             .build();
 
+    private static final AcmeClientSpi acmeClient;
+    static {
+        acmeClient = loadAcmeClient();
+    }
+
+    private static AcmeClientSpi loadAcmeClient() {
+        for (AcmeClientSpi acmeClient : ServiceLoader.load(AcmeClientSpi.class, ElytronSubsystemMessages.class.getClassLoader())) {
```

Comment:
There will only be one (WildFlyAcmeClient) but if there was more than one, it wouldn't matter because only one is needed to prove ownership of a domain name.